tools/slicec-cs/src/member_util.rs

```diff
@@ -34,20 +33,13 @@ pub fn field_declaration(field: &Field, field_type: FieldType) -> String {
         writeln!(prelude, "[{obsolete}]");
     }
 
-    // Fields inherit the visibility of their parents.
-    let mut modifiers = vec![field.parent().access_modifier()];
-
-    // Check if this field, or it's parent struct, are marked with `cs::readonly`.
-    let mut attributes = field.all_attributes().concat().into_iter();
-    if attributes.any(|a| a.downcast::<CsReadonly>().is_some()) {
-        modifiers.push("readonly");
-    }
+    let setter = if field.is_cs_readonly() { "init" } else { "set" };
```

Comment:
For simple one line computations like this, which are _only_ used by a `format`,
it's customary to just move it directly into the `format` call.